src/test/lib/QuicDrill.cpp

```diff
@@ -222,20 +222,6 @@ QuicDrillInitialPacketFailureTest(
 
     QuicAddr ServerAddress(QuicAddrFamily);
     DrillSender Sender;
-    const uint8_t Disabled = FALSE;
-    const uint8_t Enabled = TRUE;
-
-    Status =
```

Comment:
@anrossi which test cases actually needed this? Because they all continue to pass without this.

Reply:
After looking through the logs, I think disabling encryption was actually completely unnecessary, since these tests exercise the packet drop code paths that happen before decryption.

Reply:
The tests still pass because the metric for passing is coarse-grained; it can't tell if the initial packet was dropped because it decrypted wrong, or because the contents of the initial packet was incorrect.
So it's passing, but it's not actually exercising the code path we want it to.

Reply:
Every test case is for a check that happens before decryption. So, unless we have a way to expose packet drop counts, per drop reason, you'll never know. I looked at the logs and all the dropped reasons seemed good.
```
 598 [1]6CFC.1E18::2020/07/30-08:58:25.069704400 [Microsoft-Quic][test] START QuicDrillInitialPacketCids, v4/SourceCid/Invalid/Short/Invalid length
 652 [6]6CFC.72C4::2020/07/30-08:58:25.071214900 [Microsoft-Quic][bind][0x1A8010C3C20] DROP packet[0] Src=127.0.0.1:60228 Dst=127.0.0.1:60227 Reason=Client Long header Initial packet too short, 26.
 683 [3]6CFC.1E18::2020/07/30-08:58:25.182627000 [Microsoft-Quic][test] END QuicDrillInitialPacketCids
 684 [3]6CFC.1E18::2020/07/30-08:58:25.182732900 [Microsoft-Quic][test] START QuicDrillInitialPacketCids, v4/SourceCid/Invalid/Long/Valid length
 737 [4]6CFC.23B0::2020/07/30-08:58:25.184775100 [Microsoft-Quic][bind][0x1A8010C3C20] DROP packet[0] Src=127.0.0.1:60230 Dst=127.0.0.1:60229 Reason=Client Long header Initial packet too short, 40.
 769 [6]6CFC.1E18::2020/07/30-08:58:25.290682900 [Microsoft-Quic][test] END QuicDrillInitialPacketCids
 770 [6]6CFC.1E18::2020/07/30-08:58:25.290793400 [Microsoft-Quic][test] START QuicDrillInitialPacketCids, v4/SourceCid/Invalid/Long/Invalid length
 823 [0]6CFC.0BC4::2020/07/30-08:58:25.292342400 [Microsoft-Quic][bind][0x1A8010C3C20] DROP packet[0] Src=127.0.0.1:60232 Dst=127.0.0.1:60231 Reason=Greater than allowed max CID length.
 855 [0]6CFC.1E18::2020/07/30-08:58:25.397937300 [Microsoft-Quic][test] END QuicDrillInitialPacketCids
 856 [0]6CFC.1E18::2020/07/30-08:58:25.398042400 [Microsoft-Quic][test] START QuicDrillInitialPacketCids, v4/DestCid/Valid/Short/Valid length
 910 [5]6CFC.1CA4::2020/07/30-08:58:25.399658800 [Microsoft-Quic][bind][0x1A801868A20] DROP packet[0] Src=127.0.0.1:60234 Dst=127.0.0.1:60233 Reason=Client Long header Initial packet too short, 27.
 941 [4]6CFC.1E18::2020/07/30-08:58:25.506048000 [Microsoft-Quic][test] END QuicDrillInitialPacketCids
...
```

Reply:
Great news!